main/acle.md

```diff
@@ -5015,9 +5014,16 @@ depend on ZT0 as well as ZA.
 | **ZA state before A** | **ZA state after A** | **Possible if…**                       |
 | --------------------- | -------------------- | -------------------------------------- |
 | off                   | off                  | F's uses and A's clobbers are disjoint |
-| dormant               | dormant              | " " "                                  |
-| dormant               | off                  | " " ", and A clobbers `"za"`           |
-| active                | active               | F uses `"za"` and/or `"zt0"`           |
+|                       |                      | or F is an                             |
+|                       |                      | ["#agnostic-ZA"](#agnostic-za)         |
+|                       |                      | function.                              |
+| dormant               | dormant              | "" "" ""                               |
+| dormant               | off                  | A clobbers `"za"`, but F does not      |
+|                       |                      | [use](#uses-state) `"za"`or `"zt0"`.   |
+| active                | active               | F uses `"za"` and/or `"zt0"`, or       |
```

Comment:
I think it's only this row that needs to change.  The old version of the others is still correct.

I'd suggest the same “neither…nor” change as above, but it's hard to get the suggestion right in the github editor :-)